### PR TITLE
Sign Up Page UI

### PR DESCRIPTION
**Pull request by abhishek310001**
# Description:

This PR introduces the Sign up form of the Odin Project's Intermediate HTML and CSS section. From this project I've implemented the use of forms and their action.

The screenshot of UI is attached in this PR for reference

## Feature:

- The forms input fields are required such that It can't be submitted before filling them out correctly.
- Placeholder are placed such that It can give idea of filling it.
- Added Box shadow on the Form area so that it will be differentiated from other sections
- Added input box Focus property so that when it is selected it is highlighted with different color.
- The **Create Account** button does not collect any information for now.

## UI Screenshot:

sign-up-form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                            <input type="text" name="first_name" id="first_name" placeholder="John" required>
+                        </div>
+                        <div class="last-name">
+                            <div>
+                                <label for="last_name">LAST NAME</label>
+                            </div>
+                            <input type="text" name="last_name" id="last_name" placeholder="Doe" required>
+                        </div>
+                    </div>
+                    <div class="other-details">
```
Comment (devanuj97) [inline index.html]: css class name is not meaningful, name `other-details`  does not any meaningful context

**Comment by devanuj97** [comment]
@abhishek310001 list of thing i'm pointing out which can be addressed 

- Add JS validation form password match 
- Form input field look weird, please try padding and border to make it look better 
- Make text above the form section little small it looks too large

**Comment by devanuj97** [comment]
@abhishek310001 are you sure you UI looks like this
https://cdn.statically.io/gh/TheOdinProject/curriculum/5f37d43908ef92499e95a9b90fc3cc291a95014c/html_css/project-sign-up-form/sign-up-form.png

If not how did you decided it's completed ?

**Review by devanuj97 — APPROVED** [review]

**Review by devanuj97 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (abhishek310001) [reply index.html]: will change it to some meaningful context

**Pull request merged**